programs/psinode/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ target_link_libraries(psinode
     debug_eos_vm
     psibase_net
 )
+target_link_directories( psinode PUBLIC ${ICU_LIBRARY_DIR} )
```

Comment:
Since the dependency on ICU is apparently coming from Boost through `psibase`, it would be better to set this once on the `psibase` library target.